src/pages/StocktakeEditPage.js

```diff
@@ -83,24 +83,23 @@ export class StocktakeEditPage extends React.Component {
   }
 
   renderCell(key, item) {
+    const isEditable = !this.props.stocktake.isFinalised;
     switch (key) {
       default:
         return item[key];
-      case 'countedTotalQuantity':
+      case 'countedTotalQuantity': {
+        const emptyCellContents = isEditable ? '' : tableStrings.no_change;
```

Comment:
If the stocktake is finalised and there was no change, shouldn't this item be pruned anyway? Or does this cover there being no net change, but a redistribution amongst batches?

Reply:
https://github.com/sussol/mobile/pull/542/commits/598c8a26a5c648bc090e0d78445ce77175efffbc 
Also should only display No change for StocktakeBatches not for StocktakeItem (as it would be pruned now)